experiments/compositions/composition/api/v1alpha1/composition_types.go

```diff
@@ -95,6 +95,15 @@ const (
 	NamespaceModeExplicit NamespaceMode = "explicit"
 )
 
+// ReadyOn defines ready condition for a GVK
+type ReadyOn struct {
```

Comment:
is there an example of how we are using this? IIUC this is to define the condition that marks if a gvk is ready, not sure what is `progress` about?

Reply:
Meant to remove progress. Will remove it. It was deemed not well defined enough when me and @justinsb  discussed it.